core/src/components/picker-column/picker-column.md.scss

```diff
@@ -1,6 +1,8 @@
 @import "./picker-column.scss";
 @import "../../themes/ionic.globals.md";
 
-:host .picker-item-active {
+:host .picker-item-active,
+::slotted([slot="prefix"]),
+::slotted([slot="suffix"]) {
```

Comment:
This is to mirror the legacy picker, though I wasn't sure if we wanted to keep it.

Reply:
I'm fine keeping it, though my 2 cents is using the color makes me think it's interactive/selected when it is not.

Reply:
That's true 🤔 I'll revert this for now. We can always adjust it if people report it later.